lte/gateway/docker/s1ap/readme.md

```diff
@@ -15,35 +15,9 @@ s1ap/publish.sh yourregistry.com/yourrepo/
 
 # Run s1aptester
 
-1. Create a host that has 3 interfaces. eth0 which is your SGi interface, eth1 which is your S1 interface, and a third interface which is your ssh management interface to connect to the instance while doing the tests. You could skip the third interface if you have some kind of serial console access.
+1. Create an ubuntu 20.04 host that has 3 interfaces. eth1 which is your SGi interface, eth2 which is your S1 interface, and a third interface which is your ssh management interface to connect to the instance while doing the tests. You could skip the third interface if you have some kind of serial console access. Your host should have at least 20 GB of HD.
 
-Due to the IP address contraints of how s1aptester is written, we have to hardcode the addresses of eth0 and eth1. Here is an example netplan configuration:
-
-```
-network:
-    ethernets:
-        # SGi
-        eth0:
-            dhcp6: false
-            dhcp4: no
-            addresses: [192.168.60.142/24]
-            nameservers:
-              addresses: [8.8.8.8,8.8.4.4]
-        # S1
-        eth1:
-            dhcp6: false
-            dhcp4: no
-            addresses: [192.168.129.1/24]
-            nameservers:
-              addresses: [8.8.8.8,8.8.4.4]
-        # SSH Management
-        eth2:
-            dhcp4: true
-            dhcp6: false
-    version: 2
-```
-
-2. Move into AGW docker directory on the host and run start script. Make sure that your `.env` file points to your registry with your AGW and s1aptester images.
+2. [Deploy a containerized AGW](https://github.com/magma/magma/tree/master/lte/gateway/docker), move into AGW docker directory `/var/opt/magma/docker` on the host and run start script. Make sure that your `.env` file points to your registry with your AGW and s1aptester images.
```

Comment:
- I am unsure I understand this correctly. What is meant?
`**To** deploy a containerized AGW, move into ...` _or_ does one first have to deploy the containerized AGW (How?) and then move into the directory?
- `run _the_ start script`
- Which one is the start script? Can we explicitly name it, please, e.g. "Run the start script `my-fancy-start-script.sh`.
- Can we give an example here, how the `.env` file would need to look using a freshly installed Ubuntu machine with the local docker registry, please.

Reply:
Clarified the deployment steps.